File: 20508_Jayanthi JAVA Assignments/day2Assignments/src/com/sonata/Student.java
package com.sonata;

public class Student
{
	int StdId;
	String StdName;
	int StdCls;
	Student(int id,String name,int cls)
	{
		this.StdId=id;
		this.StdName=name;
		this.StdCls=cls;
	}
public void display()
{
	System.out.println(StdId);
	System.out.println(StdName);
	System.out.println(StdCls);
}
public static void main(String args[])
{
	Student s1=new Student(20508,"jayanthi",28);
	s1.display();
	Student s2=new Student(20,"jayanthi",8);
	s2.display();
}
}

File: TSEXAMPLE/Student.ts
class Student
{
	StdId:number;
	StdName:string;
	StdCls:number;
	constructor(id:number,name:string,cls:number)
	{
		this.StdId=id;
		this.StdName=name;
		this.StdCls=cls;
	}
display()
{
	console.log(this.StdId);
    console.log(this.StdName);
    console.log(this.StdCls);
	}
}
	
var e1=new Student(234,"kutti",459);
var e2=new Student(22,"jayanthi",097);
e1.display();
e2.display();
